src/domain/project/index.ts

```diff
@@ -72,6 +74,7 @@ const validateTalkProject = (talkProject: LatestProjectType["talk"]) => {
   }
 };
 
+// TODO: マイグレーション（とファイルの最初のeslint-disable）を別ファイルに移す
 /**
  * プロジェクトファイルのマイグレーション
  */
```

Comment:
（関係ないけど）
こうするとコメントまとまって良いかも
```suggestion
/**
 * プロジェクトファイルのマイグレーション
 * TODO: マイグレーション（とファイルの最初のeslint-disable）を別ファイルに移す
 */
```

Reply:
個人的にはモジュールの説明とモジュールのTODOは分離すべきだと思ってるのでこうしてます（ファイルの説明はそのファイル以外に出す必要はない、みたいな）

Reply:
迷いどころですが、これたぶんファイルの最初の`eslint-disable`の下に書くのが合ってるかも。
あるいは区間disableにしたらTODOコメントとすごく近くなりそう。

でもまあ別に良さそう！！･･･気が向いたらで！！
ということでresolve！